aws/table_aws_dms_replication_instance.go

```diff
@@ -91,7 +92,6 @@ func tableAwsDmsReplicationInstance(_ context.Context) *plugin.Table {
 				Name:        "multi_az",
 				Description: "Specifies whether the replication instance is a Multi-AZ deployment.",
 				Type:        proto.ColumnType_BOOL,
-				Transform:   transform.FromField("MultiAZ"),
 			},
```

Comment:
why we are removing this transform? This is required.

Reply:
resolved